andela/mmdp_site
# MMDP-174 Implement CRUD for thematic pillars

**What does this PR do?**

This PR implements create, read, update and delete functionality for thematic pillars.

**Description of Task to be completed?**

As an administrator, I want to be able to add content to be displayed on the 'Thematic Pillars' page
As an administrator, I want to be able to edit content displayed on the 'Thematic Pillars' page
As an administrator, I want to be able to view content on the 'Thematic Pillars' page
As an administrator, I want to be able to archive content displayed on the 'Thematic Pillars' page

**How should this be manually tested?**

Git clone https://github.com/andela/mmdp_site
Check out  ft-thematic-pillars-MMDP-174 branch using git.
Run yarn run: server to start the server.
Test API Endpoints.

**Any background context you want to provide?**

None

**What are the relevant Jira issues?**

[MMDP-174](https://viisaus.atlassian.net/browse/MMDP-174)

**Screenshots (if appropriate)**


**Questions:**

None
